doc/getting_started/getting_started.rst

```diff
@@ -20,14 +20,42 @@ Follow one of the following guides for your host operating system.
    macOS <installation_mac.rst>
    Windows <installation_win.rst>
 
-Clone the Zephyr Repository
-***************************
+.. _get_the_code:
+
+Get the source code
+*******************
+
+In order to get the source code for the Zephyr Project you need to use
```

Comment:
Using the word 'need' communicates that it is not possible to manually clone the git repo's, which I assume is not so.

Reply:
How about:

    Zephyr's multi-purpose :ref:west` tool lets you easily get the Zephyr project source code,
    instead of manually cloning the Zephyr repos.